scene/gui/split_container.h

```diff
@@ -79,15 +79,15 @@ class SplitContainer : public Container {
 		Ref<Texture2D> grabber_icon_v;
 	} theme_cache;
 
-	Control *_getch(int p_idx) const;
-
 	Ref<Texture2D> _get_grabber_icon() const;
 	void _compute_middle_sep(bool p_clamp);
 	void _resort();
 
 protected:
 	bool is_fixed = false;
 
+	Control *_get_split(int p_idx) const;
```

Comment:
I'd call it `get_containable_child` or something like that. This is a method we probably want to end up exposing on `Container` itself as a public getter, since fetching fitting children is something all containers always do.